src/helpers/TrafficDetailsHelper.tsx.ts

```diff
@@ -0,0 +1,27 @@
+import { authentication } from '../utils/Authentication';
```

Comment:
src/helpers/TrafficDetailsHelper.tsx.ts ??
Why do we need this extra .tsx.ts ?

Reply:
Sorry... My mistake. I fixed the naming.